ext/js/comm/anki-connect.js

```diff
@@ -655,6 +666,45 @@ export class AnkiConnect {
                 fields: fields2,
                 modelName,
                 cards: cards2,
+                cardsInfo: [],
+            };
+            result2.push(item2);
+        }
+        return result2;
+    }
+
+    /**
```

Comment:
Can you add a description for what this function does? The logic is somewhat hard to grok from glimpsing

Reply:
Added.